src/ApiRequestPanel.js

```diff
@@ -528,7 +533,7 @@ export class ApiRequestPanel extends AmfHelperMixin(
   }
 
   render() {
-    if (this._isNonWebApi()) {
+    if (this._isNonWebApi() && !this.forceRender) {
```

Comment:
This was originally designed to stop rendering when a fragment is loaded. This can easily be done in the method/endpoint documentation just not to render the request panel when the incompatible model was provided which would render this unnecessary. However, why do you need this to be forced? What is the use case?

Reply:
When a partial model is supplied to the request panel, it cannot determine whether or not the model belongs to a WebAPI or an AsyncAPI. Therefore the behavior of this component not rendering content needs to be overridden